docker-compose.yml

```diff
@@ -43,13 +39,14 @@ services:
       - db-configure
     env_file:
       - .env
+    platform: linux/amd64
     image: node:18.12.1-alpine
```

Comment:
@RageZBla I understand @theyokohamalife was able to get this working locally by running `docker rmi node:18.12.1-alpine` on her machine, but I still see the node alpine image listed here. Can you please check this so we have a repeatable docker build?